instrumentation-appender-api/src/main/java/io/opentelemetry/instrumentation/appender/api/DelegatingLogEmitterProvider.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.appender.api;
+
+import io.opentelemetry.sdk.logs.SdkLogEmitterProvider;
+
+public final class DelegatingLogEmitterProvider implements LogEmitterProvider {
```

Comment:
In the agent these `Delegating...` classes will have to be loaded by the agent classloader (the api classes will be loaded by bootstrap), right? Does it make sense to at least put them into a separate package? (a separate module would probably be "cleaner", but that may be a bit too much for library instrumentations)

Reply:
oh no, this is absolutely right!

> a separate module would probably be "cleaner", but that may be a bit too much for library instrumentations

I went the separate module route so we don't have to do any weird shadow configuration for now, but I'll add this to the list of things to revisit.